app/models/miq_report/generator.rb

```diff
@@ -245,18 +245,25 @@ def generate_performance_results(options = {})
     results
   end
 
+  def performance_report_time_range
+    if db_options[:specific_date]
```

Comment:
What do you think about renaming this option to `custom_rnge` or `custom_time_range`? I think that would be more descriptive

Reply:
@lpichler Sorry, I was referring to the option `db_options[:specific_date]` when I suggested renaming it. I think `performance_report_time_range` was fine since it covers both cases.

What do you think about putting the method name back to `performance_report_time_range` and renaming the option to `db_options[:custom_time_range]`?

Reply:
oh, yes sure, I missed correct line , thanks that you noticed that 👍 

now it _should_ be done 👍